tests/mpi/periodicity_01.cc

```diff
@@ -158,7 +159,6 @@ namespace Step40
                                               0,
                                               Functions::ZeroFunction<dim>(),
                                               constraints);
-    DoFTools::make_hanging_node_constraints (dof_handler, constraints);
```

Comment:
should we not enforce this somehow from `make_hanging_node_constraints`? or at least mention it in documentation?